### ISSUE TITLE
Test support with Sidekiq 6

### ISSUE THREAD

**Issue by wenderjean** [davydovanton/sidekiq-statistic#157]

**Comment by wenderjean** [comment]
@kostadriano If I'm not wrong you were running on Rails 6, right? Did you get any issues?

**Comment by kostadriano** [comment]
Yes @wenderjean , I had a problem using `sidekiq 5.2` and `rails 6.0.3` the workers wasn't being registered. But with `sidekiq 6.0.7` it worked fine.

**Comment by gabrielbaldao** [comment]
I tried to reproduce this bug, using two cases, I use a old personal project, and after called @adriano, 
I use the same versions used by him, as cited below, and in another scenario I used the last version from `sidekiq` 
in both cases I cannot reproduct the bug, everything worked as expected.
Probably some update already fix this case.
```console
	gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
	gem 'sidekiq', '~> 5.2', '>= 5.2.7'
	gem 'sidekiq-statistic'
```
```console
	$ bundle exec sidekiq --version
	#Sidekiq 5.2.9

	$ bundle exec rails --version
	#Rails 6.0.3.2
```
![primeira screen](https://user-images.githubusercontent.com/20587352/91082443-0de97600-e61f-11ea-9704-d2a6d5f6a18b.png)
First Case

```console
	gem 'rails', '>= 6.0.3.2'
	gem 'sidekiq'
	gem 'sidekiq-statistic'
```

```console
	$ bundle exec sidekiq --version
	#Sidekiq 6.1.1

	$ bundle exec rails --version
	#Rails 6.0.3.2
```
![segunda screen](https://user-images.githubusercontent.com/20587352/91082462-1477ed80-e61f-11ea-9e9e-bfd10ecbcc5b.png)
Second case

**Comment by rbUUbr** [comment]
Started to use it with `sidekiq 6.1.3` will post here some updates on next week

**Comment by rbUUbr** [comment]
jfyi: still everything is ok
